modules/mobile/app/controllers/mobile/v0/appointments_controller.rb

```diff
@@ -26,6 +26,7 @@ def cancel
       end
 
       def create
+        # not seeing this. probably because this endpoint isn't in use
```

Comment:
Category: I am here with supporting debugging info
Action: remove. if we want to log this data, it should happen on failures only, not on successes.